satellite/admin/server.go

```diff
@@ -68,6 +68,7 @@ func NewServer(log *zap.Logger, listener net.Listener, db DB, config Config) *Se
 
 	// When adding new options, also update README.md
 	server.mux.HandleFunc("/api/user", server.addUser).Methods("POST")
+	server.mux.HandleFunc("/api/user/{useremail}", server.updateUser).Methods("PUT")
```

Comment:
I've been actually wondering whether we should have the api as `/api/user/{user-id}` for consistency. And then something separate `GET /api/users/findByEmail?email={email}`. But that's a separate PR.